scripts/ci/test_automation.sh

```diff
@@ -44,11 +44,8 @@ then
 fi
 
 target_profile=${AZURE_CLI_TEST_TARGET_PROFILE:-latest}
-if [ "$target_profile" = "2017-03-09" ]; then
-    # example: 2017-03-09-profile
-    target_profile=$target_profile-profile
-elif [ "$target_profile" = "2018-03-01" ]
-then
+if [ "$target_profile" != "latest" ]; then
+    # example: 2019-03-01-hybrid
     target_profile=$target_profile-hybrid
```

Comment:
we will need a change here when we introduce profiles for Government clouds, they won't end with -hybrid, they will be ending with -profile..

Reply:
Its ok to keep like this for now